composer.json

```diff
@@ -16,12 +16,12 @@
         "php": ">=5.3.3",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "guzzle/guzzle": "^2.8|^3.0",
+        "guzzle/guzzle": "^2.8||^3.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.4|^3.0",
-        "symfony/console": "^2.1|^3.0",
-        "symfony/stopwatch": "^2.2|^3.0",
-        "symfony/yaml": "^2.1|^3.0"
+        "symfony/config": "^2.4||^3.0",
```

Comment:
I'm sure according to the Composer documentation, but I agree we should have one change in PR. I just remove it.